app/templates/components/event-invoice/event-info.hbs

```diff
@@ -12,9 +12,9 @@
   <div class="ui padded segment">
     <strong>{{t 'At'}} {{this.event.locationName}}</strong>
     <br>
-    <strong>{{t 'From'}}:</strong> {{header-date this.event.startsAt}}
+    <strong>{{t 'From'}}:</strong> {{general-date this.event.startsAt}}
```

Comment:
I think header date used a different format than general-date?

Reply:
what we can do is that wherever header-format is being used, we can manually pass format as param

Reply:
it uses ```'dddd, MMMM Do YYYY, h:mm A'``` either we need to little modify this format and match it with the standardized formats or use this only.

Reply:
Exactly

Reply:
use this only?

Reply:
match it with standard please